src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -38,8 +40,11 @@ public IntegrityCheck(BibDatabaseContext bibDatabaseContext) {
             return result;
         }
 
-        result.addAll(new AuthorNameChecker("author").check(entry));
-        result.addAll(new AuthorNameChecker("editor").check(entry));
+        for (String fieldName : entry.getFieldNames()) {
+            if (InternalBibtexFields.getFieldExtras(fieldName).contains(BibtexSingleFieldProperties.PERSON_NAMES)) {
```

Comment:
One way to enhance readability would obviously be to have shorter class names. I honestly think that it is the major concern here. `Internal.get(field).contains(NAME)` is pretty much more readable... However, in this case, moving to AuthorNameChecker clearly makes more sense and is probably what I had done with more insight into the Checkers. I'll give it a go.